discovery-provider/src/queries/get_undisbursed_challenges.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, TypedDict, Optional
+from sqlalchemy import and_, asc
+from src.models import UserChallenge, Challenge, ChallengeDisbursement
+
+
+class ChallengeResponse(TypedDict):
```

Comment:
I think we have this guy in two files now? Maybe we could just import?

Reply:
They are different, can rename